docs/book/src/reference/providers.md

```diff
@@ -9,6 +9,7 @@ updated info about which API version they are supporting.
 - [EKS](https://github.com/kubernetes-sigs/cluster-api-provider-aws/tree/main/bootstrap/eks)
 - [Kubeadm](https://github.com/kubernetes-sigs/cluster-api/tree/main/bootstrap/kubeadm)
 - [Talos](https://github.com/siderolabs/cluster-api-bootstrap-provider-talos)
+- [MicroK8s](https://github.com/canonical/cluster-api-bootstrap-provider-microk8s)
```

Comment:
We should also add a Control plane section here to cover the micro K8s control plane provider 🙂

Reply:
Do you want me to add it in this PR or on a separate one?

We should put there:
```
kubeadm
microk8s
nested
talos
```

Reply:
We can do it in this PR if it's okay for you - otherwise a follow up is fine. Just seems like the list is starting to get long enough to add here, and I've seen a lot of movement in the community on control plane providers.

Reply:
Added a Control Plane section with the respective providers mentioned in alphabetical order.

Reply:
Thx!